susemanager-utils/testing/docker/scripts/reset_pgsql_database.sh

```diff
@@ -13,7 +13,8 @@ echo $PERLLIB
 ./build-schema.sh
 
 export SYSTEMD_NO_WRAP=1
-sysctl -w kernel.shmmax=18446744073709551615
+# Won't work: read-only file system
+# sysctl -w kernel.shmmax=18446744073709551615
```

Comment:
To what extent is it OK to comment that? Isn't that somehow needed for pgsql?

Reply:
Unfortunately I don't remember what this was needed for exactly. For now I will remove efforts to run unit tests from this PR and focus on checkstyle only. Will create a new PR about the unit tests soon and the topic will be back.

Reply:
This part will become relevant and will be revisited when we want to run the unit tests in a workflow.